packages/entity-browser/tests/test-bundler.js

```diff
@@ -1,4 +1,8 @@
+import Enzyme from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
```

Comment:
I think we should move this import to a separate file (maybe package `tocco-test-util`) and import this file in every test-bundler? I reckon we won't remain with `enzyme-adapter-react-16` once React 17 is here and we don't want to update each test-bundler.js every time React is updated.
```
import Adapter from 'enzyme-adapter-react-16'

export default Adapter
```